### DISCUSSION TITLE
not able to integrate airflow with slack ,getting error:-Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6e3b7075b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))

### DISCUSSION THREAD

**Discussion by rajans163** [apache/airflow#25194] [Q&A]
When i am trying to call slack via slackwebhookoperator getting the error

requests.exceptions.ConnectionError: HTTPConnectionPool(host='https', port=80): Max retries exceeded with url: //hooks.slack.com/services/T03Q0F8ANJZ/B03Q3GJG2CB/*** (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6e3b7075b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))

**Comment by alexkruc** [comment]
I think that something is wrong with your Slack connection or how you provide the connection string to the operator..
According to the message - see this part:
> Max retries exceeded with url: //hooks.slack.com/services/T03Q0F8ANJZ/B03Q3GJG2CB/***

This URL is malformed.. It needs to start with `http` or `https`.. Please check how you define the connection and such :) I suggest looking at the code examples of the Slack operator and debug on how it pulls the connection and go according to it -- it will help you to understand what's going on.
